docs/open-source-tizen/tools/gbs/gbs-devel.md

```diff
@@ -1,92 +1,85 @@
-
-
 # gbs devel
 
-## Name
+Use the `gbs devel` subcommand to facilitate package maintainers in maintaining the packaging branch in the orphan-packaging branch model. The command offers the following actions:
 
-gbs devel - Facilitates package maintainers to better maintain packaging branch in the orphan-packaging branch model by offering the following actions:
+- `start` creates a new development branch with the upstream version embedded in its name.
+- `export` exports patches from a development branch to its corresponding packaging branch.
+- `drop` removes an unnecessary development branch.
+- `switch` switches between the packaging branch and the corresponding development branch.
+- `convert` converts a package to the orphan-packaging branch model.
 
-- start - creates new development branch with upstream version embedded in its name.
-- export - exports patches from a development branch to its corresponding packaging branch.
-- drop - removes unnecessary development branch.
-- switch - switches between packaging branch and the corresponding development branch.
-- convert - converts a package to the orphan-packaging branch model.
+For more information on the 2 maintenance models (orphan-packaging and joint-packaging) and their differences, see [Maintenance Models Supported by GBS](../../porting/maintenance-models.md).
 
-For the difference between the two maintenance models, orphan-packaging and joint-packaging, refer to [Maintenance Models Supported by GBS](https://source.tizen.org/documentation/reference/git-build-system/maintenance-models-supported-gbs).
+For command usage details, enter:
+
+```bash
+gbs devel help
+```
 
-An indicative example of workflow is shown below:
+The following code shows an example workflow:
 
 ```bash
 # maintainer: create upstream branch and packaging branch (initial packaging)
 ...
 # maintainer: push packaging and upstream branches (and tags) to Git/Gerrit
 $ git push --tags origin upstream master
- 
+
 # maintainer: start development branch
 $ gbs devel start
- 
+
 # maintainer: push development branch to Git/Gerrit
 $ git push origin development/master/1.0
- 
+
 # developer: clone package
 $ gbs clone git://review.tizen.org/example.git
- 
+
 # developer: modify code, test, commit on development branch
 $ ...
 $ gbs build
- 
+
 # developer: push changes for review
 $ git push origin development/master/1.0:refs/for/development/master/1.0
- 
+
 # maintainer: after review, merge changes in Gerrit
 # maintainer: generate patches, update packaging/release branch
 $ gbs devel export
 $ git add .
 $ git commit -m"New change"
- 
+
 # maintainer: push packaging branch for review
 $ git push master:refs/for/master
- 
+
 # maintainer: once merged, submit for integration
 $ gbs submit
 ```
 
-## Synopsis
-
-- Subcommmand
-
-  ```bash
-  gbs devel <Second_Level_Subcommand> [Options]
-  ```
-
-- Second Level Subcommands
+## Action Commands
```

Comment:
The term 'synopsis' is  used for command manual.
please see https://git-scm.com/docs/git.

Most of gbs command manual missed synopsis. 

The following is the latest synopsis of `gbs devel`:
```bash
gbs devel [-h] [--packaging-dir PACKAGING_DIR] [--spec SPEC]
                 [--upstream-tag UPSTREAM_TAG] [--retain-history]
                 [gitdir] {start,export,drop,switch,convert}
```
I wanted to add synopsis in all gbs manual pages but there was no time to update them.

Please keep the base docs, if they have the synopsis.

Reply:
OK, we will change the term to synopsis. And add the synopsis to the git devel topic.

It would be best to have it in all the command topics. Could you list the missing ones here, so we can add them?